habitat/sims/habitat_simulator/habitat_simulator.py

```diff
@@ -225,18 +238,39 @@ def create_sim_config(
         overwrite_config(
             config_from=self.habitat_config.HABITAT_SIM_V0,
             config_to=sim_config,
+            ignore_keys={"gpu_gpu"},
```

Comment:
Can we add some comments to the keys like : hablab only field /  deprecated field or etc.

Reply:
Where though? In the default? In all the configs?

Reply:
Probably just here.  A comment at to why we are ignoring that key